app/views/layout/html_head.html.php

```diff
@@ -1,8 +1,7 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta name="description" content="<?php echo $GLOBALS["viewables"]["meta-description"] ?>">
-<meta name="keywords" content="<?php echo $GLOBALS["viewables"]["meta-keywords"] ?>">
```

Comment:
It was that `meta-keywords` wasn't, I added description [here](https://github.com/umple-ucosp/umpr.ui/pull/18/files#diff-8fda5307a067372337ec4b9adb16b55fR6)